### ISSUE TITLE
RSeqC junction_saturation.py bug

### ISSUE THREAD

**Issue by shurjo** [ewels/MultiQC#475]
```
============================================================
Module rseqc raised an exception: Traceback (most recent call last):
  File "/usr/local/Anaconda/envs_app/multiqc/0.9/lib/python3.5/site-packages/multiqc-0.9-py3.5.egg/EGG-INFO/scripts/multiqc", line 344, in multiqc
    output = mod()
  File "/usr/local/Anaconda/envs_app/multiqc/0.9/lib/python3.5/site-packages/multiqc-0.9-py3.5.egg/multiqc/modules/rseqc/rseqc.py", line 65, in __init__
    n['junction_saturation'] = junction_saturation.parse_reports(self)
  File "/usr/local/Anaconda/envs_app/multiqc/0.9/lib/python3.5/site-packages/multiqc-0.9-py3.5.egg/multiqc/modules/rseqc/junction_saturation.py", line 57, in parse_reports
    total = self.junction_saturation_all[s_name].values()[-1]
TypeError: 'odict_values' object does not support indexing
============================================================
```

**Comment by ewels** [comment]
Thanks for the bug report! Do you have a file that I can use to replicate the error please?

Phil

**Comment by ewels** [comment]
Also - I just noticed that it looks like you're running MultiQC v0.9. Do you still get the same error if you upgrade to version 1.0?

**Comment by ewels** [comment]
To confirm, this was reported in #400 and fixed in 6c10943d1526d2c450151dd0ecd6e060e875b0a0

If you update to the latest version of MultiQC you should be fine. Let me know if you run into any problems @shurjo 

Phil
